### PR TITLE
Code Highlight markup and Link colouration fix constants.adoc

### PR DESCRIPTION
**Pull request by animeshsrivastava24**
Code Highlight markup and Link colouration fix
Ref #546 
@per1234  Please review does this matches with what you mentioned in the comment https://github.com/arduino/reference-en/issues/546#issuecomment-509971175

### CODE REVIEW

File: Language/Variables/Constants/constants.adoc [hunk 1]
```diff
@@ -38,16 +38,16 @@ When reading or writing to a digital pin there are only two possible values a pi
...
+A pin may also be configured as an INPUT with link:../../../functions/digital-io/pinmode[`pinMode()`], and subsequently made HIGH with `link:../../../functions/digital-io/digitalwrite[digitalWrite()]`. This will enable the internal 20K pullup resistors, which will _pull up_ the input pin to a `HIGH` reading unless it is pulled `LOW` by external circuitry. This can done alternatively by passing `INPUT_PULLUP` as argument to the link:../../../functions/digital-io/pinmode[`pinMode()`] funtion, as explained in more detail in the section "Defining Digital Pins modes: INPUT, INPUT_PULLUP, and OUTPUT" further below.
 [%hardbreaks]
 
-When a pin is configured to OUTPUT with `pinMode()`, and set to `HIGH` with `digitalWrite()`, the pin is at:
+When a pin is configured to OUTPUT with link:../../../functions/digital-io/pinmode[`pinMode()`], and set to `HIGH` with `digitalWrite()`, the pin is at:
```
Comment (per1234) [inline Language/Variables/Constants/constants.adoc]: No big deal, but if everything else is going to be linkified, this one might as well be too.
```suggestion
When a pin is configured to OUTPUT with link:../../../functions/digital-io/pinmode[`pinMode()`], and set to `HIGH` with link:../../../functions/digital-io/digitalwrite[`digitalWrite()`], the pin is at:
```

Comment (animeshsrivastava24) [reply Language/Variables/Constants/constants.adoc]: Resolved it, please review.

**Comment by per1234** [comment]
Thanks!

**Comment by SeppPenner** [comment]
@per1234 Again a question on this: I somehow remeber that you told me once not to include links into the backticks because the links are already highlighted as links (Not as code). Or did I misunderstand something there?

**Comment by per1234** [comment]
@SeppPenner you're referring to https://github.com/arduino/reference-en/issues/546. Luckily, @animeshsrivastava24 found a solution to that. Previously, we were trying to use this style of markup to linkify code words with backticks:
```
link:../../../functions/digital-io/pinmode[`pinMode()`]
```
The problem was that for some reason this caused the link coloration of the text to be suppressed when the content was rendered on arduino.cc, so you could only tell that text was a link by hovering the mouse pointer over it. For this reason, when we wanted to make a link obvious we had to avoid using backticks on the code words:
```
link:../../../functions/digital-io/pinmode[pinMode()]
```
Which was unfortunate. @animeshsrivastava24 discovered that if you use this markup:
```
`link:../../../functions/digital-io/pinmode[pinMode()]`
```
then you get the code highlighting and link coloration both, like this:
[`pinMode()`]()

My idea is that we can use both styles of markup, as I explain in detail here:
https://github.com/arduino/reference-en/issues/546#issuecomment-509971175

**Comment by SeppPenner** [comment]
> Previously, we were trying to use this style of markup to linkify code words with backticks:

I can remeber that, yeah. It didn't work as expected.

> Which was unfortunate. @animeshsrivastava24 discovered that if you use this markup:

> `link:../../../functions/digital-io/pinmode[pinMode()]`
> then you get the code highlighting and link coloration both, like this:
> pinMode()

Good to know, I just wanted to be sure that this is correct.

Your idea from https://github.com/arduino/reference-en/issues/546#issuecomment-509971175 seems the best way to go for this. I agree with you that multiple occurences of links within a certain amount of text will make it probably unreadable as you stated there. That was something I didn't think about when I created https://github.com/arduino/reference-en/issues/462.

**Pull request merged**